.github/workflows/ci.yml

```diff
@@ -82,11 +84,11 @@ jobs:
       with:
         path: ${{ github.workspace }}/install/deps
         # Including ${{ runner.temp }} is a workaround taken from https://github.com/robotology/whole-body-estimators/pull/62 to fix macos configuration failure on https://github.com/dic-iit/bipedal-locomotion-framework/pull/45
-        key: source-deps-${{ runner.os }}-${{runner.temp}}-vcpkg-robotology-${{ env.vcpkg_robotology_TAG }}-ycm-${{ env.YCM_TAG }}-yarp-${{ env.YARP_TAG }}-iDynTree-${{ env.iDynTree_TAG }}-catch2-${{ env.Catch2_TAG }}
+        key: source-deps-${{ runner.os }}-${{runner.temp}}-vcpkg-robotology-${{ env.vcpkg_robotology_TAG }}-ycm-${{ env.YCM_TAG }}-yarp-${{ env.YARP_TAG }}-iDynTree-${{ env.iDynTree_TAG }}-catch2-${{ env.Catch2_TAG }}-qhull-${{ env.Qhull_TAG }}
 
 
     - name: Source-based Dependencies [Windows]
-      if: steps.cache-source-deps.outputs.cache-hit != 'true' && matrix.os == 'windows-latest'
+      if: matrix.os == 'windows-latest'
```

Comment:
Why this if was removed?

Reply:
I did it for forcing the compilation of the dependencies on windows. Unfortunately here https://github.com/dic-iit/bipedal-locomotion-framework/pull/51/commits/7d5aeee669c1cb38d598991223f15d52f2f49805  the compilation went fine without considering `qhull`. 
Since now we can use `qhull` on windows the only ways for checking through the GitHub action were:
1. Removing the cache (I don't know if it is possible)
2. Avoid checking if the cache key is found.

Since I wasn't able to invalidate the cache I opted for the second approach. Do you know if it's possible to clean the cache?